src/main/java/org/terasology/climateConditions/HyperthermiaComponent.java

```diff
@@ -23,4 +23,41 @@
     public float regenMultiplier = 0.8f;
     public float maxHealthMultiplier = 0.8f;
     public float thirstMultiplier = 2f;
+    public float hyperthermiaLevel;
```

Comment:
This can simply be called `level` - that is is about hyperthermia is given by the surrounding class anyways.
```suggestion
    public float level;
```

Reply:
Are there any guarantees we can give about the level? Is it always greater or equal to 1? Can it increase indefinitely? Are negative levels possible?

Reply:
Well, for now only the values 1,2,3 are supported.. I'll add that in a comment.